shared/test/integration/require_from_app.js

```diff
@@ -0,0 +1,10 @@
+var appRoot = require('app-root-path');
+
+// TODO Export this helper in its own npm package
+module.exports = function(app, resource) {
```

Comment:
Just a thought, you can probably use ```app``` (communications) and optional ```entryPoint``` (dialer/contacts) as it's in reality. But since it will go away in the future, I don't really care.